README.md

```diff
@@ -0,0 +1,13 @@
+# IJoke (PAC1)
+
+## Why?
+
+We want to provide a convenient, modern-looking web application that allows users to read, add, and like jokes.
+
+## What?
+
+## How?
+
```

Comment:
Add information about what technologies we are going to use

Reply:
Good input! I forgot